app/views/works/_edit_javascript.html.erb

```diff
@@ -134,18 +200,12 @@
       var num = incrementCounter("#creator_count");
       addCreatorHtml(num, "", "", "");
     } else {
-      // Display the initial list of creators.
-      //
-      // Eventually we will need to render them in a predefined
-      // order but we don't have order data availble yet.
-      $(".creator-data").each(function(ix, el) {
-        var num = $(el).data("num");
-        var orcid = $(el).data("orcid");
-        var givenName = $(el).data("given-name");
-        var familyName = $(el).data("family-name");
-        var sequence = $(el).data("sequence");
-        addCreatorHtml(num, orcid, givenName, familyName, sequence);
-      });
+      // Adds the existing creators making sure we honor the ordering.
+      var creators = sortedCreators();
```

Comment:
Makes sure they are rendered based on their sequence on the database. They were rendering in order without this code but I wasn't sure they were always going to render in order so I decided to force them to be in order just to be sure.